### PR TITLE
-xstartup option added

### PR DESCRIPTION
**Pull request by Siot**
This gives the ability to load different xstartup file for each VNC server.

### CODE REVIEW

File: unix/vncserver [hunk 1]
```diff
@@ -337,20 +341,20 @@ if (-e "/tmp/.X11-unix/X$displayNumber" ||
 }
 $ENV{VNCDESKTOP}= $desktopName;
 
-system($exedir."vncconfig -iconic >> " . &quotedString($desktopLog) . " 2>&1 &");
+system($exedir."vncconfig -nowin >> " . &quotedString($desktopLog) . " 2>&1 &");
```
Comment (hildred) [inline unix/vncserver]: good idea but completely unrelated to the rest of this patch.

File: unix/vncserver.man [hunk 2]
```diff
@@ -123,6 +125,11 @@ this has the effect of terminating Xvnc when the user logs out of the window
 manager.
 .
 .TP
+.B \-xstartup \fIfilename\fP
+Name of the shell script file, located at %HOME/.vnc/, specifying X applications 
+to be run when the VNC desktop is started. Defaults to %HOME/.vnc/xstartup.
+.
+.TP
```
Comment (hildred) [inline unix/vncserver.man]: This could be worded better, but other than that everything looks good.

**Comment by dcommander** [comment]
I would suggest maybe adopting the -xstartup and -noxstartup options used by TurboVNC:

https://github.com/TurboVNC/turbovnc/blob/master/unix/vncserver.in

They are more flexible, since you can specify any arbitrary script instead of just one in ~/.vnc.

File: unix/vncserver (hunk 1 above)
Comment (Siot) [inline unix/vncserver]: Oops, this was involuntary.

**Comment by Siot** [comment]
I will take a look to TurboVNC script. Thanks! (And learn a little of perl)

**Comment by Siot** [comment]
Adopted TurboVNC options.

**Comment by CendioOssman** [comment]
Please merge 9539e51d93f7f1df39b455a98cd72b55e0488fc3, d5e719b12d196570b0155ead93a86a4e692c5244, 42a39cfc0d27e361be45b24f776885e20a5ebbca and 227e53865a1dc65ac4d526c6bcc620964d4a9abb.

79dd42feeafa6398b909c7b13f9ed18745986e1d and 220d948f384595a60485d474f40624eb18fb3303 should also be mostly merged in to that, except for the `-noxstartup` stuff which can be kept as a separate commit.

42481bdf658c44287e26a5be6dc4394234add32c should be merged into whatever commit caused the problem, and 2bd8650db1a9b37679fc589592a293e94b1e4422 should be merged with the other `-noxstartup` stuff.

So rebase everything and just force an update of this branch. github will update the PR, so no need to create another just because you are shuffling things around.

**Comment by Siot** [comment]
Ok, I was updating files throught github web interface. I'm not sure how to do this. Let me investigate a little.

**Comment by CendioOssman** [comment]
Getting there. :)
It would still be nice if `-noxstartup` could be a separate commit. And the current commit message is just a mess.

**Comment by CendioOssman** [comment]
Looks good. Thanks for all your work. :)

**Comment by CendioOssman** [comment]
I'm sorry. I can't merge this. You've based your worked of the 1.5 branch. I'm going to have to ask you to rebase things on the master branch.

**Comment by CendioOssman** [comment]
I see you opened #226 for `-noxstartup`, but what happened to the other commit?

**Comment by Siot** [comment]
Well, I'm not sure, how many times I will have to rebase something... then, I'm "ensuring shooting" (literal translation of local expression). Once -noxstartup added, I will send another PR based on latest master branch code with -xstartup option. I think this will result in less time waste due my inexperience with git.

**Comment by CendioOssman** [comment]
Alright. Just checking that it wasn't overlooked. :)

**Pull request closed without merging**